deploy/00_resolve_keep_token.js

```diff
@@ -0,0 +1,22 @@
+module.exports = async ({ getNamedAccounts, deployments }) => {
+  const keepTokenAddress = process.env.KEEP_TOKEN_ADDRESS
+  const { deploy, save, log } = deployments
+  const { deployer } = await getNamedAccounts()
+
+  if (keepTokenAddress) {
+    log(`using externally provided KeepToken address ${keepTokenAddress}`)
+
+    // Save as simple deployment just to make it accessible for next scripts.
+    await save("KeepToken", { address: keepTokenAddress })
+  } else {
```

Comment:
What do you think about adding a network check? We shouldn't deploy stubs for networks other than local (e.g. ropsten, mainnet). We should fail for non-local if the external address is missing.

Reply:
Makes sense. What about the `UniswapV2Router` eg. for the `ropsten` or any other test network? Should we deploy a stub or use a real instance?

Reply:
Let's use real instances on public testnets, if possible.